src/@next/components/atoms/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+import * as S from "./styles";
+import { IFormError, IProps } from "./types";
+
+export const ErrorMessage: React.FC<IProps> = ({ errors }: IProps) =>
+  !!(errors && errors.length) && (
+    <S.ErrorMessage>
+      {errors.map((error: IFormError) => error.message).join(" ")}
```

Comment:
`error: IFormError` this should be auto-typed based on errors prop, no need to be explicit here